pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java

```diff
@@ -413,6 +447,32 @@ public void run() {
     return executor.awaitTermination(1, TimeUnit.HOURS);
   }
 
+  private boolean verifySegment(File indexDir) {
+    File localTempDir = null;
+    try {
+      try {
+        localTempDir = new File("segmentVerification", org.apache.pinot.common.utils.FileUtils.getRandomFileName());
```

Comment:
Use `FileUtils.getTempDirectory()` as the parent temp dir?

Reply:
done